index.js

```diff
@@ -79,12 +80,11 @@
 			if ( latitude > 180 ) latitude = latitude - (2 ** 32) * SCALAR_E7;
 			if ( longitude > 180 ) longitude = longitude - (2 ** 32) * SCALAR_E7;
 
-			if ( type === 'json' ) latlngs.push( [ latitude, longitude ] );
```

Comment:
https://github.com/theopolisme/location-history-visualizer/pull/34 really fixes it